shared/common-adapters/standard-screen.native.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import {Component} from 'react'
+import type {Props} from './standard-screen'
+
+class StandardScreen extends Component<void, Props, void> {
```

Comment:
@awendland I think either ticket an implementation of it, or if you're sure that there's no native equivalent, just make the import fail on native so we don't end up getting null components rendered there by accident